app/assets/javascripts/models/beta/note.js

```diff
@@ -0,0 +1,32 @@
+import httpService from '../../services/httpService';
+import changeCase from 'change-object-case';
+
+export const destroy = (projectId, storyId, noteId) => {
```

Comment:
the `() => { return expression }` for a body with a single expression can be reduced to just `() => expression`, so you may do `(projectId, storyId, noteId) => httpService
     .delete(`/projects/${projectId}/stories/${storyId}/notes/${noteId}`)
     .catch(error => console.error(error))`